### PR TITLE
Added home page template

### PR DESCRIPTION
**Pull request by rerobika**

### CODE REVIEW

**Comment by rerobika** [comment]
cc: @nandormagyar

File: src/main/webapp/WEB-INF/layouts/default.jsp [hunk 1]
```diff
@@ -24,10 +24,24 @@
     <!-- Bootswatch flatly theme -->
     <link rel="stylesheet" href="https://bootswatch.com/flatly/bootstrap.min.css">
 
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+
+    <!-- Include all compiled plugins (below), or include individual files as needed -->
+    <script src="${contextRoot}/js/bootstrap.js"></script>
```
Comment (nandor-magyar) [inline src/main/webapp/WEB-INF/layouts/default.jsp]: Bootstrap is already loaded using CDN, see couple of lines below.

**Review by nandor-magyar — CHANGES_REQUESTED** [review]
It is cool doe. >)

File: src/main/webapp/WEB-INF/layouts/default.jsp (hunk 1 above)
Comment (rerobika) [reply src/main/webapp/WEB-INF/layouts/default.jsp]: That's true I've missed that part.

**Pull request merged**